docs/architecture/cloud-native/candidate-apps.md

```diff
@@ -1,52 +1,57 @@
 ---
 title: Candidate Apps for Cloud Native
```

Comment:
```suggestion
title: Candidate apps for cloud native
```

Reply:
Not sure if you want to use Cloud Native or cloud native, so feel free to modify my headings suggestions a bit